<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1099</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(206, 137, 103);
background-color: rgb(85, 85, 127);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>991</width>
      <height>631</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">alternate-background-color: rgb(255, 170, 255);
background-color: rgb(255,170, 255);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Gestion Personnel</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>441</width>
        <height>251</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color:rgb(255, 0, 127);</string>
      </property>
      <property name="title">
       <string>Tri</string>
      </property>
      <widget class="QTableView" name="tableViewT">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>58</y>
         <width>321</width>
         <height>171</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(255, 225, 255);</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton">
       <property name="geometry">
        <rect>
         <x>29</x>
         <y>13</y>
         <width>51</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>CIN</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_salaire">
       <property name="geometry">
        <rect>
         <x>86</x>
         <y>13</y>
         <width>76</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Salaire</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="geometry">
        <rect>
         <x>168</x>
         <y>13</y>
         <width>60</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>340</y>
        <width>521</width>
        <height>251</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color:rgb(255, 0, 127);</string>
      </property>
      <property name="title">
       <string>RECHERCHE</string>
      </property>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>40</y>
         <width>226</width>
         <height>122</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="0">
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 225, 255);</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLineEdit" name="lineEdit">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 225, 255);</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="lineEdit_3">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 225, 255);</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="checkBox_2">
          <property name="text">
           <string>Prénom</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="checkBox_3">
          <property name="text">
           <string>Mobile</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>Nom</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>180</y>
         <width>181</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 255);</string>
       </property>
       <property name="text">
        <string>Valider</string>
       </property>
      </widget>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>251</width>
         <height>201</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 225, 255);</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>10</y>
        <width>521</width>
        <height>319</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color:rgb(255, 0, 127);</string>
      </property>
      <property name="title">
       <string>Afficher Personnel</string>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>280</y>
         <width>341</width>
         <height>32</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="pushButton_4">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 170, 255);</string>
          </property>
          <property name="text">
           <string>Supprimer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_6">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 170, 255);</string>
          </property>
          <property name="text">
           <string>Statistique</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pushButton_10">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>280</y>
         <width>121</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 255);</string>
       </property>
       <property name="text">
        <string>Créer un pdf</string>
       </property>
      </widget>
      <widget class="QTableView" name="tableView_3">
       <property name="geometry">
        <rect>
         <x>48</x>
         <y>30</y>
         <width>381</width>
         <height>239</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 127);
background-color: rgb(255, 225, 255);</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>441</width>
        <height>319</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color:rgb(255, 0, 127);</string>
      </property>
      <property name="title">
       <string>Ajouter Personnel</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>40</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>CIN</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>73</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditnom">
       <property name="geometry">
        <rect>
         <x>132</x>
         <y>73</y>
         <width>171</width>
         <height>27</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 255);
background-color: rgb(255, 225, 255);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditcin">
       <property name="geometry">
        <rect>
         <x>132</x>
         <y>40</y>
         <width>171</width>
         <height>27</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 255);
background-color: rgb(255, 225, 255);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditprenom">
       <property name="geometry">
        <rect>
         <x>132</x>
         <y>106</y>
         <width>171</width>
         <height>27</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 255);
background-color: rgb(255, 225, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>106</y>
         <width>58</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Prénom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>139</y>
         <width>50</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Mobile</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditmobile">
       <property name="geometry">
        <rect>
         <x>132</x>
         <y>139</y>
         <width>171</width>
         <height>27</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 255);
background-color: rgb(255, 225, 255);
</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>172</y>
         <width>96</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Rôle</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEditdate_naissance">
       <property name="geometry">
        <rect>
         <x>132</x>
         <y>205</y>
         <width>112</width>
         <height>25</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 255);
background-color: rgb(255, 225, 255);
</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>205</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Date_Naissance</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>236</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Salaire</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditsalaire">
       <property name="geometry">
        <rect>
         <x>132</x>
         <y>236</y>
         <width>171</width>
         <height>27</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 255);
background-color: rgb(255, 225, 255);</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>132</x>
         <y>172</y>
         <width>187</width>
         <height>27</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 85, 127);</string>
       </property>
       <item>
        <property name="text">
         <string>Finance</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ressources Humaines</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Administration</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Informatique</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Marketing</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>270</y>
         <width>93</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 255);</string>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>270</y>
         <width>93</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 255);</string>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>270</y>
         <width>93</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 255);</string>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Gestion Rôle</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>10</y>
        <width>471</width>
        <height>581</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color:rgb(255, 0, 127);
</string>
      </property>
      <property name="title">
       <string>Afficher Rôle</string>
      </property>
      <widget class="QTableView" name="tableViewD">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>391</width>
         <height>411</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 255);
background-color: rgb(255, 225, 255);</string>
       </property>
      </widget>
      <widget class="QPushButton" name="supprimer_role">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>500</y>
         <width>93</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 255);</string>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>481</width>
        <height>581</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color:rgb(255, 0, 127);</string>
      </property>
      <property name="title">
       <string>Ajouter Rôle</string>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>32</x>
         <y>150</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 11pt &quot;Palatino Linotype&quot;;
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>220</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_description">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>220</y>
         <width>221</width>
         <height>27</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 255);
background-color: rgb(255, 225, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>38</x>
         <y>300</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Nb_Employés</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nb_employes">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>300</y>
         <width>221</width>
         <height>27</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 255);
background-color: rgb(255, 225, 255);</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>140</y>
         <width>221</width>
         <height>27</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 85, 127);</string>
       </property>
       <item>
        <property name="text">
         <string>Finance</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Marketing</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ressources Humaines</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Administration</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Informatique</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="ajouter2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>440</y>
         <width>93</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 255);</string>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_8">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>440</y>
         <width>93</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 255);</string>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
      <widget class="QPushButton" name="modifier_role">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>440</y>
         <width>93</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 255);</string>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1099</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
